frappe/public/js/frappe/ui/like.js

```diff
@@ -32,32 +26,31 @@ frappe.ui.toggle_like = function ($btn, doctype, name, callback) {
 			// renable click
 			$btn.css("pointer-events", "auto");
 
-			if (!r.exc) {
-				for (const like of document.querySelectorAll(".like-action")) {
-					if (like.dataset.name === name && like.dataset.doctype === doctype) {
-						like.classList.toggle("not-liked", add === "No");
-						like.classList.toggle("liked", add === "Yes");
-					}
-				}
```

Comment:
This segment was used to update *like* icons everywhere in the DOM, not just the clicked button. The like icon is shown in the form view and the list view, thus **I think* that liking a document opened in the form would **not** update the like icon in the list view, and the other way too. Right?

Reply:
List and form view are not in the DOM at the same time. If you add a like from the list view, then switch to the form view (and vice versa), the like will still be visible.

Reply:
https://github.com/frappe/frappe/assets/14891507/2d2ec8f5-9849-4c9f-9658-c86a5d8b24de